hyp3_rtc_gamma/check_coreg.py

```diff
@@ -11,159 +11,157 @@
+        elif os.path.isdir("geo"):
+            mlog = "geo/{}".format(myfile)
+        else:
+            raise CoregistrationError(f"Can't find {myfile}")
+
+        a = np.zeros(6)
+        r = np.zeros(6)
+
+        with open(mlog, "r") as g:
+            for line in g:
+                if 'final range offset poly. coeff.:' in line:
+                    tmp = re.split(":", line)
+                    vals = tmp[1].split()
+                    if len(vals) == 1:
+                        r[0] = float(vals[0])
+                        logging.info(f"Range offset is {r}")
+                    elif len(vals) == 3:
+                        r[0] = float(vals[0])
+                        r[1] = float(vals[1])
+                        r[2] = float(vals[2])
+                        logging.info(f"Range polynomial is {r}")
+                    elif len(vals) == 6:
+                        r[0] = float(vals[0])
+                        r[1] = float(vals[1])
+                        r[2] = float(vals[2])
+                        r[3] = float(vals[3])
+                        r[4] = float(vals[4])
+                        r[5] = float(vals[5])
+                        logging.info(f"Range polynomial is {r}")
+
+                if 'final azimuth offset poly. coeff.:' in line:
+                    tmp = re.split(":", line)
+                    vals = tmp[1].split()
+                    if len(vals) == 1:
+                        a[0] = float(vals[0])
+                        logging.info(f"Azimuth offset is {a}")
+                    elif len(vals) == 3:
+                        a[0] = float(vals[0])
+                        a[1] = float(vals[1])
+                        a[2] = float(vals[2])
+                        logging.info(f"Azimuth polynomial is {a}")
+                    elif len(vals) == 6:
+                        a[0] = float(vals[0])
+                        a[1] = float(vals[1])
+                        a[2] = float(vals[2])
+                        a[3] = float(vals[3])
+                        a[4] = float(vals[4])
+                        a[5] = float(vals[5])
+                        logging.info(f"Azimuth polynomial is {a}")
+
+                if 'final model fit std. dev. (samples) range:' in line:
+                    tmp = re.split(":", line)
+                    vals = tmp[1].split()
+                    rng_error = float(vals[0])
+                    val = tmp[2].strip()
+                    azi_error = float(val)
+                    logging.info(f"Range std dev: {rng_error}  Azimuth std dev: {azi_error}")
+                    error = np.sqrt(rng_error * rng_error + azi_error * azi_error)
+                    logging.info(f"error is {error}")
+                    if error > max_error:
+                        logging.info("error > max_error")
+                        logging.info("std dev is too high, using dead reckoning")
+                        logging.info("Granule failed coregistration")
```

Comment:
`logging.warning` might be called for here

Reply:
Good call -- `display` was limited to either `logging.info` or `logging.error`, so the switch to the local context logger would allow us to do that.